lib/rubocop/ast/node/mixin/method_dispatch_node.rb

```diff
@@ -8,6 +8,8 @@ module MethodDispatchNode
       extend NodePattern::Macros
       include MethodIdentifierPredicates
 
+      ACCESS_MODIFIERS = %i[public protected private module_function].freeze
```

Comment:
@backus: There's a bit of a backlog on the new 2.1 style modifiers. Most of our indentation cops don't handle them at this point. 😞 

That said, I can add the differentiating predicates in when I work on those. No need to get it in with this PR, since it's not related. 🙂